packages/notebook/src/windowing.ts

```diff
@@ -71,6 +71,21 @@ export class NotebookViewModel extends WindowedListModel {
  * Windowed list layout for the notebook.
  */
 export class NotebookWindowedLayout extends WindowedLayout {
+  private _header: Widget | null = null;
+
+  public get header(): Widget | null {
+    return this._header;
+  }
+  public set header(v: Widget | null) {
```

Comment:
Could you add some documentation please?

```suggestion
  get header(): Widget | null {
    return this._header;
  }
  set header(v: Widget | null) {
```